modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase

```diff
@@ -240,4 +240,78 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-173512. Verify that If no claim was made, the message ‘You can submit up to 2 claims’ appears"
+	@priority = "5"
+	test VerifyMessageWhenNoClaimWasMade {
+		task ("Given I go to add a new MDF Request") {
+			PRMMarketing.gotoAddNewMDF();
+		}
+
+		task ("When I fill all mandatory fields on the MDF Request") {
+			PRMMDFRequest.fillGoalsPage(
+				accountName = "Company Name 1",
+				additionalOptions = "6.x to DXP Migration/Upgrade,Migration from competitor platform",
+				campaignDescription = "Campaign Description",
+				campaignName = "Campaign Name",
+				countryName = "Global",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("And I add a new activity") {
+			PRMMDFRequest.newActivity(
+				activityDescription = "Activity Description",
+				activityName = "Activity Name 1",
+				budgetList = "Sponsorship Fee-2000,Room Rental-3000",
+				followUpStrategy = "test",
+				leadFollowUpList = "Other",
+				leadList = "No",
+				tactic = "Other",
+				targetOfLeads = "test",
+				typeOfActivity = "Miscellaneous Marketing");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("When I submit the MDF") {
+			Button.click(button = "Continue");
+
+			Button.click(button = "Submit");
+		}
+
+		task ("And I go to MDF Details") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
+
+			PRMUtils.gotoPRM();
+
+			PRMHomePage.gotoQuickLinksOption(
+				buttonLink = "mdf-requests",
+				buttonText = "MDF");
+
+		}
+
+		task ("And I approve the MDF Request") {
+			Click(
+				key_text = "Request-",
+				locator1 = "Link#ANY");
```

Comment:
Hi @heiditrindade, this is not a good idea, since you aren't specifying the name of the request, and just a prefix... You need a path that get the MDF Name and click on Request-ID that contains this name. It could be `//thead//th/p[text()='Name']//ancestor::table//tbody//th//span[text()='${key_MDFName}']//ancestor::tr//th[1]`